app/cdap/components/Administration/TetheringTabContent/ActionPopover/index.tsx

```diff
@@ -86,10 +86,14 @@ const ActionsPopover = ({ target, onDeleteClick, onEditClick }: IActionsPopoverP
       showPopover={false}
     >
       <ul>
-        <ListItem disabled={!canEdit} onClick={onEditClick}>
-          {T.translate(`${PREFIX}.Actions.edit`)}
-        </ListItem>
-        <hr />
+        {canEdit && (
```

Comment:
It was recommended to hide the `Edit` option until we provide functionality for it.